web-animations-2/Overview.bs

```diff
@@ -2353,6 +2376,152 @@ Items sorted earlier are executed before those sorted later.
 
 </ins>
 
+Animation Triggers {#triggers}
+-----------------
+
+### Overview ### {#triggers-overview}
+
+An <dfn export>animation trigger</dfn> is used to control the playback
+of its associated [=animation=] for time-driven animations.
+Like animations, [=animation triggers=] are associated with a [=timeline=]
+and are attached to an [=animation attachment range|attachment range=].
+
+Issue: Should there be any effect of triggers on scroll-driven animations?
+
+### Animation Trigger Active Interval ### {#trigger-active-interval}
+
+[=Animation triggers=] define an <dfn for="animation trigger">active interval</dfn>,
+which is the period of time during which the trigger is scheduled to produce its effect
+on its associated [=animation=]’s playback.
+Each [=animation trigger=] has only one such interval,
+which is defined by its [=default range=] and [=exit range=].
+
+### Animation Trigger State ### {#trigger-state}
+
+An [=animation trigger=] |trigger| with [=animation trigger type=] |type|
+and an associated [=animation=] |animation|
+has an internal <dfn export lt="animation trigger state">state</dfn> that can be one of the following:
+
+<dl dfn-for="animation trigger state" dfn-type=value>
+  <dt><dfn>idle</dfn>
+  <dd>
+    The [=animation effect=] associated |animation| remains in
+    its [=animation effect/before phase=] and stays at zero [=animation/start time=].
+
+  <dt><dfn>primary</dfn>
+  <dd>
+    The ''primary'' effect defined by |type| is applied to |animation|.
```

Comment:
There's a bit of a mismatch between this and the text in css-animations-2 where the text for single-animation-trigger-type specifies when the effect is applied. The text above implies that the effect is applied continually while in the state, but the text in single-animation-trigger-type implies that the effect is applied at a discrete moment in time. If all of the effects are discrete then I think it would be simpler to have the effect explicitly only invoked when the trigger's state changes.

Reply:
I've tried making that more explicit in the text, LMK if that's good